index.html

```diff
@@ -4,11 +4,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Document </title>
     <link rel="stylesheet" href="style.css"> 
 </head>
 <body>
-    <h1>Hello  Microverse </h1>
+    <h1>Hello Microverse </h1>
 
     
```

Comment:
## 5.
### Your Readme file
![README](https://user-images.githubusercontent.com/93039359/184926591-aa23667a-df14-4fed-a877-fefc8a6811f7.png)

### Add these things
![missing-readme](https://user-images.githubusercontent.com/93039359/184926813-00b307fa-bb38-4283-bc57-bc50b4026468.png)

- [ ] Please add missing things in your readme file. :+1: This is your readme means documentation of your project so please take it seriously :+1: